src/components/AccountDropdown/AccountDropdown.tsx

```diff
@@ -217,7 +219,10 @@ export const AccountDropdown: FC<AccountDropdownProps> = ({
             <AccountChildOption
               key={`${iterAccountId}-${index}`}
               title={makeTitle(iterAccountId)}
-              cryptoBalance={fromBaseUnit(accountBalances[iterAccountId][assetId], asset.precision)}
+              cryptoBalance={fromBaseUnit(
+                accountBalances?.[iterAccountId]?.[assetId] ?? 0,
```

Comment:
this kinda scares me - i expect there will be other places that we run into this problem where we expect account data to exist and it doesn't